backend/src/main/java/com/google/sps/servlets/FindRecipesServlet.java

```diff
@@ -26,22 +34,35 @@
 import javax.servlet.http.HttpServletResponse;
 
 /* Servlet that:
- * in Post request, returns a list of recommended recipes based on the ingredients in the request
+ * in Post request returns web links based on the inputted ingredients
  */
 @WebServlet("/api/find-recipes")
 public class FindRecipesServlet extends HttpServlet {
+  private static final int MAX_NUMBER_OF_INGREDIENTS = 3;
 
-  // TODO: based on the ingredients in the request, retrieve 5 links to scrape recipes from
-  // now: returns a list of recipes scraped from bbc good food
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String ingredients = request.getReader().readLine();
+    String json = Jsoup.connect(
+      "https://customsearch.googleapis.com/customsearch/v1?cx=c318350d7878a8a31&q="
+        + ingredients +"&key=" + ApiKeys.customSearchKey).ignoreContentType(true).execute().body();
```

Comment:
Make the link into a constant to improve readability or create a function getCustomSearchLink() that returns the link